.github/scripts/code_style.sh

```diff
@@ -30,56 +30,58 @@ if ! [ -x "$(command -v uncrustify)" ]; then
-    ./demos/sample_azure_iot/*.c                              \
-    ./demos/sample_azure_iot_adu/*.c                          \
-    ./demos/sample_azure_iot_adu/*.h                          \
-    ./demos/sample_azure_iot_ca_recovery/*.c                  \
-    ./demos/sample_azure_iot_pnp/*.c                          \
-    ./demos/sample_azure_iot_pnp/*.h                          \
-    ./demos/sample_azure_iot_gsg/*.c                          \
-    ./demos/sample_azure_iot_gsg/*.h                          \
-    ./demos/common/transport/*.c                              \
-    ./demos/common/transport/*.h                              \
-    ./demos/common/utilities/*.c                              \
-    ./demos/common/utilities/*.h                              \
-    ./demos/projects/ESPRESSIF/adu/port/*.c                   \
-    ./demos/projects/ESPRESSIF/adu/port/*.h                   \
-    ./demos/projects/ESPRESSIF/adu/main/*.c                   \
-    ./demos/projects/ESPRESSIF/az-nvs-cert-bundle/main/*.c    \
-    ./demos/projects/ST/b-l475e-iot01a/port/*.c               \
-    ./demos/projects/ST/b-l475e-iot01a/port/*.h               \
-    ./demos/projects/ST/b-l475e-iot01a/config/*.h             \
-    ./demos/projects/ST/stm32h745i-disco/cm7/port/*.c         \
-    ./demos/projects/ST/stm32h745i-disco/cm7/port/*.h         \
+    uncrustify -c ./uncrustify.cfg --no-backup --replace                                    \
+    ./demos/sample_azure_iot/*.c                                                            \
+    ./demos/sample_azure_iot_adu/*.c                                                        \
+    ./demos/sample_azure_iot_adu/*.h                                                        \
+    ./demos/sample_azure_iot_ca_recovery/*.c                                                \
+    ./demos/sample_azure_iot_pnp/*.c                                                        \
+    ./demos/sample_azure_iot_pnp/*.h                                                        \
+    ./demos/sample_azure_iot_gsg/*.c                                                        \
+    ./demos/sample_azure_iot_gsg/*.h                                                        \
+    ./demos/common/transport/*.c                                                            \
+    ./demos/common/transport/*.h                                                            \
+    ./demos/common/utilities/*.c                                                            \
+    ./demos/common/utilities/*.h                                                            \
+    ./demos/projects/ESPRESSIF/adu/port/*.c                                                 \
+    ./demos/projects/ESPRESSIF/adu/port/*.h                                                 \
+    ./demos/projects/ESPRESSIF/adu/main/*.c                                                 \
+    ./demos/projects/ESPRESSIF/az-ca-recovery/components/sample-azure-iot-ca-recovery/*.c   \
```

Comment:
do you really want to put the word "sample" on this directory?

Reply:
at least here, its for legacy reasons. We can look to change them all if we want.